homeassistant/auth/auth_store.py

```diff
@@ -414,6 +433,14 @@ async def async_load(self) -> None:
             else:
                 last_used_at = None
 
+            if (
+                expire_at := rt_dict.get("expire_at")
+            ) is None and token_type == models.TOKEN_TYPE_NORMAL:
+                if last_used_at:
+                    expire_at = last_used_at.timestamp() + REFRESH_TOKEN_EXPIRATION
+                else:
+                    expire_at = now_ts + REFRESH_TOKEN_EXPIRATION
```

Comment:
It would be good to cover this line before merging since its critical functionality